### PR TITLE
Lock closed issues/prs

### PR DESCRIPTION
**Pull request by hoxbro**
Same as the one used on Lumen https://github.com/holoviz/lumen/blob/main/.github/workflows/lock.yaml

For now, it is set to run every hour, every day, until the backlog is completed, at which point I will change it to run each Tuesday night.

### CODE REVIEW

File: .github/workflows/lock.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v5
+        with:
+          issue-inactive-days: "100"
```
Comment (maximlt) [inline .github/workflows/lock.yaml:22]: Out of curiosity, where does 100 come from?

**Review by maximlt — APPROVED** [review]
LGTM

I think HoloViews is going to be the real test for this as Lumen doesn't have real users yet and not a deep Github history.

File: .github/workflows/lock.yaml (hunk 1 above)
Comment (philippjfr) [reply .github/workflows/lock.yaml:22]: 👍

File: .github/workflows/lock.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: "Lock Threads"
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+      # - cron: "0 0 * * 2"
```
[the remark this answers is not in the document]
Comment (hoxbro) [reply .github/workflows/lock.yaml:6]: I will change it to the commented-out cron job when the current backlog of closed issues/prs has been closed.

[the remark this answers is not in the document]
Comment (ahuang11) [reply .github/workflows/lock.yaml:6]: Ah got it. I missed your description.

**Comment by hoxbro** [comment]
I will merge this tomorrow if there are no objections.

**Pull request merged**